src/Admin.js

```diff
@@ -9,9 +9,9 @@ import adminReducer from './reducer';
 import crudSaga from './sideEffect/saga';
 import CrudRoute from './CrudRoute';
 import Layout from './mui/layout/Layout';
-import withAppTitle from './mui/layout/withAppTitle';
+import withAppTitleAndTheme from './mui/layout/withAppTitleAndTheme';
```

Comment:
this is starting to look more and more like a generic `withProps` HOC. I suggest you rename it and move it ip the directory structure.